pyqir-parser/pyqir/parser/_parser.py

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
+from tokenize import String
```

Comment:
@swernli Is this left over from the previous iteration?

Reply:
It might be... I'll remove it and commit if that doesn't break anything.